lib/fastlane/actions/slack.rb

```diff
@@ -27,11 +27,11 @@ def self.run(options)
 
         notifier = Slack::Notifier.new(options[:slack_url])
 
-        if options[:username].to_s.length > 0
-          notifier.username = options[:username]
-        else
-          notifier.username = 'fastlane'
-        end
+        notifier.username = if options[:username].to_s.length > 0
```

Comment:
Checking for this rule seems to be new to `rubocop` `0.36.0`. I'll look into disabling and discouraging this rule in `0.36.0` branch, in this PR I just discarded all changes like `var = if ...`, all that's left is fixing alignment and spaces issues.